Day-I-Social-Card/social-card/src/App.js

```diff
@@ -1,22 +1,15 @@
 import React from 'react';
 import './App.css';
+import CardContainer from './components/CardComponents/CardContainer';
+import MyHeader from './components/HeaderComponents/HeaderContainer';
+import MyFooter from './components/FooterComponents/Footer';
 
 const App = () => {
   return (
     <div>
```

Comment:
Since you're not doing anything w/ this div (Like using it for styling) and just using it because you need a wrapper, this would be a perfect candidate for a `Fragment`. So you could either change the div on line 9 to `<React.Fragment>` or import the fragment at the top on line 1 `import React, { Fragment} from 'react'` and just change the div on line 9 to `<Fragment>`. It would ultimately just lead to 1 less div in your code after it's rendered.